build.sbt

```diff
@@ -153,7 +153,8 @@ lazy val V = new {
   val gradleBloop = bloop
   val mavenBloop = bloop
   val scalafmt = "2.0.1"
-  val mdoc = "2.0.0"
+  val mdoc = "2.0.1+1-dcd42d72-SNAPSHOT"
```

Comment:
Pending new release of mdoc after https://github.com/scalameta/mdoc/pull/208 is merged